toolchain/check/context.h

```diff
@@ -363,6 +363,9 @@ class Context {
     inst_blocks().Set(SemIR::InstBlockId::Exports, exports_);
   }
 
+  // Finalizes the initialization function (__global_init).
+  auto FinalizeInits() -> void;
```

Comment:
For FinalizeInits/PushInit/PopInit, could you please rename them using Globalnit similar to the block rename?

Reply:
Done.